### PR TITLE
Milestone estimated readiness schema parameter added

### LINKED ISSUE #31: Milestone estimated readiness

### PR DESCRIPTION
**Pull request by Daniilmass**
Resolves #31

### CODE REVIEW

File: swagger.yaml [hunk 1]
```diff
@@ -1536,6 +1536,9 @@ components:
           format: date
           type: string
           description: Original end date of the milestone
+        estimatedReadiness:
+          type: string
```
Comment (kaeline) [inline swagger.yaml]: Is it really just a string? Not a date?

Comment (Daniilmass) [reply swagger.yaml]: Screenshot 2024-05-27 at 9 51 20
It is that percentage value that is hand defined by the Admin / Project owner

Comment (Daniilmass) [reply swagger.yaml]: The same schema parameter in Task had string, so I assumed it is not different for the Milestone

Comment (Jdallos) [reply swagger.yaml]: I have set both milestone and task estimated readiness as a number. There was some mention yesterday of it also being a float? Is this necessary, I would think a number 1 - 100 is suitable here?

File: swagger.yaml [hunk 2]
```diff
@@ -1581,7 +1584,7 @@ components:
           type: string
           description: Estimated duration of the task
         estimatedReadiness:
-          type: string
+          type: number
           description: Estimated readiness of the task
```
Comment (kaeline) [inline swagger.yaml:1598]: Add a comment with its range?
I think swagger ha the keywords for it https://swagger.io/docs/specification/data-models/data-types/

File: swagger.yaml (hunk 1 above)
Comment (kaeline) [reply swagger.yaml]: It probably depends just on the UI requirements, float was mentioned by @Daniilmass

Comment (Jdallos) [reply swagger.yaml]: After discussion estimatedReadiness as integer is fine.

File: swagger.yaml (hunk 2 above)
Comment (Jdallos) [reply swagger.yaml:1598]: Range added

File: swagger.yaml [hunk 3]
```diff
@@ -1536,6 +1536,14 @@ components:
           format: date
           type: string
           description: Original end date of the milestone
+        estimatedReadiness:
+          type: number
```
Comment (kaeline) [inline swagger.yaml]:
```suggestion
          type: integer
          format: int32
```

File: swagger.yaml [hunk 4]
```diff
@@ -1578,10 +1586,13 @@ components:
...
           description: User role suggested for the task
         estimatedDuration:
-          type: string
-          description: Estimated duration of the task
+          type: number
+          format: float
+          description: Estimated duration of the task in days
         estimatedReadiness:
-          type: string
+          type: number
```
Comment (kaeline) [inline swagger.yaml]:
```suggestion
          type: integer
          format: int32
```

**Review by kaeline — APPROVED** [review]

**Pull request merged**